nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/field/ObjectTimestampFieldConverter.java

```diff
@@ -39,7 +57,80 @@ class ObjectTimestampFieldConverter implements FieldConverter<Object, Timestamp>
      */
     @Override
     public Timestamp convertField(final Object field, final Optional<String> pattern, final String name) {
```

Comment:
The multiple `return` statements in this method make it much more difficult to follow. Refactoring to have a single return would be helpful.

Reply:
I followed the pattern I saw in `ObjectLocalDateTimeFieldConverter` which has multiple returns.

Reply:
Good point, scratch that recommendation.